.gitlab-ci.yml

```diff
@@ -0,0 +1,126 @@
+    - apt-get install -y wget unzip
+
+    # Setup some environment variables
+    - if [ "$CI_EXTERNAL_PULL_REQUEST_IID" != "" ]; then export PULL_REQUEST="true"; else export PULL_REQUEST="false"; fi
+    - export COMMIT_RANGE="$CI_COMMIT_BEFORE_SHA..$CI_COMMIT_SHA"
+    - export JOB_NUMBER="$CI_JOB_ID"
+    - export HOST_SRC_DIR=$CI_PROJECT_DIR
+    - export CACHE_DIR=$CI_PROJECT_DIR/cache
+    - echo PULL_REQUEST=$PULL_REQUEST COMMIT_RANGE=$COMMIT_RANGE HOST_SRC_DIR=$HOST_SRC_DIR CACHE_DIR=$CACHE_DIR
+    - echo "Commit log:" && git log --format=fuller -1
+
+    # Init cache
+    - mkdir -p $CACHE_DIR
+    - >
+      if [ "$CI_COMMIT_REF_SLUG" != "develop" -a "$CI_COMMIT_TAG" == "" ]; then
+        if [ ! -d $CACHE_DIR/ccache ]; then
+          echo "Downloading cache from develop branch"
+          if wget -O cache-artifact.zip https://gitlab.com/$CI_PROJECT_NAMESPACE/$CI_PROJECT_NAME/-/jobs/artifacts/develop/download?job=$CI_JOB_NAME; then
+            unzip cache-artifact.zip
+            rm cache-artifact.zip
+            mv cache-artifact/* $CACHE_DIR/
+          else
+            echo "Failed to download cache"
+          fi
+        else
+          echo "Not touching cache (was initialized from previous build)"
+        fi
+      else
+        echo "Not touching cache (building develop branch or tag)"
+      fi
+    # Create missing cache dirs
+    - mkdir -p $CACHE_DIR/ccache && mkdir -p $CACHE_DIR/depends && mkdir -p $CACHE_DIR/sdk-sources
+    # Keep this as it makes caching related debugging easier
+    - ls -lah $CACHE_DIR && ls -lah $CACHE_DIR/depends && ls -lah $CACHE_DIR/ccache
+
+    # Build dash_hash
+    - git clone https://github.com/dashpay/dash_hash
+    - cd dash_hash && python3 setup.py install
+
+    # Install build target specific packages (most should be installed already due to the builder image)
+    - echo PACKAGES=$PACKAGES
+    - if [ -n "$DPKG_ADD_ARCH" ]; then dpkg --add-architecture "$DPKG_ADD_ARCH" ; fi
+    - if [ -n "$PACKAGES" ]; then apt-get update && apt-get install -y --no-install-recommends --no-upgrade $PACKAGES; fi
+
+    # Make mingw use correct threading libraries
+    - update-alternatives --set i686-w64-mingw32-gcc /usr/bin/i686-w64-mingw32-gcc-posix || true
+    - update-alternatives --set i686-w64-mingw32-g++  /usr/bin/i686-w64-mingw32-g++-posix || true
+    - update-alternatives --set x86_64-w64-mingw32-gcc  /usr/bin/x86_64-w64-mingw32-gcc-posix || true
+    - update-alternatives --set x86_64-w64-mingw32-g++  /usr/bin/x86_64-w64-mingw32-g++-posix || true
+
+  script:
+    - export BUILD_TARGET="$CI_JOB_NAME"
+    - cd $CI_PROJECT_DIR
+    - ./ci/build_depends.sh
+    - ./ci/build_src.sh
+    - ./ci/test_unittests.sh
+    - ./ci/test_integrationtests.sh
+    # Copy all cache files into cache-artifact so that they get uploaded. We only do this for develop so that artifacts
+    # stay minimal for PRs and branches (we never need them)
+    - >
+      mkdir $CI_PROJECT_DIR/cache-artifact
```

Comment:
```suggestion
      mkdir $CI_PROJECT_DIR/cache-artifact;
```

Reply:
Changed the multiline command to use `|` instead of `>`, which honors newlines